### PR TITLE
ashisoft.dff version 8.1.0.1

### PR DESCRIPTION
**Pull request by mqdb**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/62276)

### CODE REVIEW

File: manifests/a/ashisoft/dff/8.1.0.1/ashisoft.dff.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ashisoft.dff
+PackageVersion: 8.1.0.1
+PackageLocale: en-US
+Publisher: Ashisoft
+PackageName: Duplicate File Finder
+License: Proprietary
+Copyright: Ashisoft all rights reserved.
+ShortDescription: Duplicate File Finder Setup
```
Comment (KaranKad) [inline manifests/a/ashisoft/dff/8.1.0.1/ashisoft.dff.locale.en-US.yaml:11]:
```suggestion
ShortDescription: Duplicate File Finder is a powerful application that uses intelligent algorithms to identify all types of duplicate files.
```
Proper description of the application.

File: manifests/a/ashisoft/dff/8.1.0.1/ashisoft.dff.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ashisoft.dff
```
Comment (KaranKad) [inline manifests/a/ashisoft/dff/8.1.0.1/ashisoft.dff.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Ashisoft.DuplicateFileFinder
```
Edit filename and foldername as well.

File: manifests/a/ashisoft/dff/8.1.0.1/ashisoft.dff.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ashisoft.dff
```
Comment (KaranKad) [inline manifests/a/ashisoft/dff/8.1.0.1/ashisoft.dff.installer.yaml:4]:
```suggestion
PackageIdentifier: Ashisoft.DuplicateFileFinder
```
Edit filename and foldername as well.

File: manifests/a/ashisoft/dff/8.1.0.1/ashisoft.dff.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: ashisoft.dff
```
Comment (KaranKad) [inline manifests/a/ashisoft/dff/8.1.0.1/ashisoft.dff.yaml:4]:
```suggestion
PackageIdentifier: Ashisoft.DuplicateFileFinder
```
Edit filename and foldername as well.

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by mqdb — COMMENTED** [review]
Is there anything else? I used the shortened version as Ashisoft had.

**Comment by Trenly** [comment]
> Is there anything else? I used the shortened version as Ashisoft had.

You will need to change the name of the folders so that the manifest files are in the directory `manifests\a\Ashisoft\DuplicateFileFinder\8.1.0.1`

**Review by ItzLevvie — APPROVED** [review]

**Review by mqdb — COMMENTED** [review]
I'm okay with these changes.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
@mqdb: Looks like you need to rename the folder and file names to match the PackageIdentifier.

`ashisoft` to `Ashisoft`
`dff` to `DuplicateFileFinder`

**Comment by mqdb** [comment]
> @mqdb: Looks like you need to rename the folder and file names to match the PackageIdentifier.
> 
> `ashisoft` to `Ashisoft` `dff` to `DuplicateFileFinder`

I am not sure I see a simple way to do this. Is there somewhere I would find a file manager, or is the suggestion that I just resubmit the whole thing. The latter seems like it would be easier

**Comment by Trenly** [comment]
> > @mqdb: Looks like you need to rename the folder and file names to match the PackageIdentifier.
> > `ashisoft` to `Ashisoft` `dff` to `DuplicateFileFinder`
> 
> I am not sure I see a simple way to do this. Is there somewhere I would find a file manager, or is the suggestion that I just resubmit the whole thing. The latter seems like it would be easier

If you presss `.` while on this screen it will bring up an editor

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/62276#issuecomment-1151042918

**Pull request closed without merging**